### ISSUE TITLE
DeviantArt extended_fetch: Parse Character Tags

### ISSUE THREAD

**Issue by GoAwayNow** [hydrusnetwork/hydrus#380]
I came across a piece today which uses specialized "subject tags" to list characters in the art separately from user tags and in a more controlled manner. It's apparently not widely used, but still, it's a good metadata source, as opposed to the mess that user tags can be, so I made a quick parser for it.
`[30, 4, ["character tags", 0, [31, 3, [[[0, [51, 1, [0, "deviation", null, null, "deviation"]]], [0, [51, 1, [0, "extended", null, null, "extended"]]], [0, [51, 1, [0, "subjectTags", null, null, "subjectTags"]]], [1, null]], 1, [84, 1, [26, 1, [[51, 1, [2, "^{\"name\": \".+\", \"type\": \"character\", .+}$", null, null, "{\"name\": \"Ruby Rose (RWBY)\", \"type\": \"character\", \"url\": \"https://www.deviantart.com?q=\\\"Ruby%20Rose%20%28RWBY%29\\\"\"}"]], [55, 1, [[[9, ["^{\"name\": \"(.+?)\", .+}$", "\\1"]]], "{\"name\": \"Ruby Rose (RWBY)\", \"type\": \"character\", \"url\": \"https://www.deviantart.com?q=\\\"Ruby%20Rose%20%28RWBY%29\\\"\"}"]]]]]]], 0, false, "character"]]`

**Comment by hydrusnetwork** [comment]
Thank you, this looks great. I'll test it out and roll it into the defaults!

**Comment by hydrusnetwork** [comment]
I am sorry to be a bother, but I am looking at this now and I can't actually find a DA image with this data to parse. I looked across RWBY results amongst others, but never got the extended/subjectTags in the JSON. I am not sure if I am looking in the wrong place or if something is not lined up right. This is in the extended_fetch DA parser, right? Can you give an example URL with this info?

**Comment by GoAwayNow** [comment]
Yeah, this is for the extended_fetch parser. These types of tags aren't enforced (but what is on DeviantArt), so they end up being kind of rare. I have no idea what url I originally used as the basis for this, but I did find this one, which has a Blake Belladonna character tag:  
[Blake Belladonna - Birthday!](https://www.deviantart.com/gigamessy/art/Blake-Belladonna-Birthday-867616408)
